nf_core/lint.py

```diff
@@ -855,13 +850,24 @@ def check_actions_awsfulltest(self):
                 assert "published" in wf[True]["release"]["types"]
                 assert "push" not in wf[True]
                 assert "pull_request" not in wf[True]
+                assert "workflow_dispatch" in wf[True]
```

Comment:
Super minor: Clearer to read if the `is` / `is not` logic is grouped together..

```suggestion
                assert "workflow_dispatch" in wf[True]
                assert "push" not in wf[True]
                assert "pull_request" not in wf[True]
```